apache-commons-collections/src/test/java/org/baeldung/java/MapUtilsTest.java

```diff
@@ -0,0 +1,87 @@
+package org.baeldung.java;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.collections4.PredicateUtils;
+import org.apache.commons.collections4.TransformerUtils;
+import org.junit.Test;
+
+public class MapUtilsTest {
+
+    @Test
+    public void whenCreateMapFrom2DArray_THEN_MUST_CREATE_Map() {
```

Comment:
Tipically, we're using the style that in this case would match whenCreateMapFrom2DArray_theMapIsCreated etc. So even the than part is better when camelCased